spec/facades/movie_db_facade_spec.rb

```diff
@@ -2,16 +2,6 @@
 
 RSpec.describe MovieDbFacade do
   describe 'happy path' do
-    # let (:film) do 
```

Comment:
I removed these commented tests for now but let me know if we wanted to implement? Wasn't sure about the purpose

Reply:
Safe to remove - I was toying with the VCR refactor and didnt want to lose my ideas. But am taking a different approach now.